testing/rust/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "wasmcloud-interface-testing"
+version = "0.0.1"
```

Comment:
I know it's just testing but I think this should start at `0.1.0` to avoid accidentally just auto-upgrading into breaking changes